RecoLocalCalo/HcalRecAlgos/BuildFile.xml

```diff
@@ -14,6 +14,8 @@
 <use   name="CondFormats/DataRecord"/>
 <use   name="vdt_headers"/>
 <use   name="rootminuit2"/> 
+<use   name="cuda"/>
+<flags CUDA_FLAGS="-Xptxas -O2,-v"/>
```

Comment:
why do you need the -O2 ?
it should already be there in the default CUDA_FLAGS.

Reply:
is left over from testing different Os and what the effect would be.